src/components/EditSections/EditContactInfo/EditContactInfo.js

```diff
@@ -12,21 +12,33 @@ import {
   Col,
   Accordion,
   Headline,
+  MultiSelection,
 } from '@folio/stripes/components';
 import { AddressEditList } from '@folio/stripes/smart-components';
 
 import { toAddressTypeOptions } from '../../data/converters/address_type';
 import contactTypes from '../../data/static/contactTypes';
+import { preferredEmailCommunicationOptions } from './constants';
 
 const EditContactInfo = ({
   expanded,
   onToggle,
   accordionId,
   addressTypes,
   preferredContactTypeId,
+  preferredEmailCommunication,
   intl,
   disabled,
 }) => {
+  const [preferredEmailCommData, setPreferredEmailCommData] = useState([]);
+
+  useEffect(() => {
+    const prefEmailCommLabeledData = preferredEmailCommunicationOptions.filter(
+      option => preferredEmailCommunication.includes(option.value)
+    );
+    setPreferredEmailCommData(prefEmailCommLabeledData);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Probably a state is not required here. Could you evaluate if this can be handled by a utility function?